repair_quotation_manual_sync/models/sale_order.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2025 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    repair_sync_warning = fields.Boolean(
+        compute="_compute_repair_sync_warning", store=False
```

Comment:
```suggestion
        compute="_compute_repair_sync_warning"
```
computed fields are not stored by default, it might be redundant. the same for below ⬇️